src/txmempool.cpp

```diff
@@ -1036,6 +1039,73 @@ size_t CTxMemPool::DynamicMemoryUsage() const {
     return memusage::MallocUsage(sizeof(CTxMemPoolEntry) + 15 * sizeof(void*)) * mapTx.size() + memusage::DynamicUsage(mapNextTx) + memusage::DynamicUsage(mapDeltas) + memusage::DynamicUsage(vTxHashes) + cachedInnerUsage;
 }
 
+bool CTxMemPool::DumpMempoolPath(const fs::path& dump_path, FopenFn mockable_fopen_function, bool skip_file_commit) const
```

Comment:
I think I agree with this as well, though I'm not very familiar with this code. Since dump/load kinda straddles `CChainState` and `CTxMemPool` neither seem like a great home, so standalone makes sense to me.

Reply:
Done as of 80b303ea1a065602cf3f84d45b04dce95bfb6952